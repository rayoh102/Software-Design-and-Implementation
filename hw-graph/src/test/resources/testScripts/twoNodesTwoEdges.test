# Tests adding two nodes, two edges between the two nodes to graph.

# Create a graph
CreateGraph twoNodesTwoEdges

# Add two nodes
AddNode twoNodesTwoEdges a
AddNode twoNodesTwoEdges b

# Add two edges
AddEdge twoNodesTwoEdges a b AB
AddEdge twoNodesTwoEdges b a BA

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
ListNodes twoNodesTwoEdges
istChildren twoNodesTwoEdges a
ListChildren twoNodesTwoEdges b