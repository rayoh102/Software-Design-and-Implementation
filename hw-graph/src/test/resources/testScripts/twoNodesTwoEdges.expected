# Tests adding two nodes, two edges between the two nodes to graph.

# Create a graph
created graph twoNodesTwoEdges

# Add two nodes
added node a to twoNodesTwoEdges
added node b to twoNodesTwoEdges

# Add two edges
added edge AB from a to b in twoNodesTwoEdges
added edge BA from b to a in twoNodesTwoEdges

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
twoNodesTwoEdges contains: a b
the children of a in twoNodesTwoEdges are: b(AB)
the children of b in twoNodesTwoEdges are: a(BA)